src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -1827,61 +1839,55 @@ public function deletePropertyImmediately($path)
      */
     protected function deleteProperty($path)
     {
+        $nodePath = PathHelper::getParentPath($path);
+        $propertyName = PathHelper::getNodeName($path);
         $this->assertLoggedIn();
```

Comment:
can we keep the assertion at the top, for readability?

Reply:
updated :)